macaroonbakery/macaroon.py

```diff
@@ -226,7 +239,31 @@ def macaroon_version(bakery_version):
     return pymacaroons.MACAROON_V2
 
 
-def parse_local_location(loc):
+class ThirdPartyLocator(object):
```

Comment:
This doesn't seem quite right to me. In Go, the ThirdPartyLocator is the interface type, so in Python I'd expect it to be an abstract base class. This type seems the equivalent of ThirdPartyStore - an in-memory implementation of ThirdPartyLocator.

Reply:
done.